### ISSUE TITLE
Auth error: Python error: init_fs_encoding: failed to get the Python codec of the filesystem encoding

### ISSUE THREAD

**Issue by Whoops** [aptakube/aptakube#101]
I have a Cluster in Oracle Cloud I'm attempting to try Aptakube on. I'm using the AppImage on Arch Linux.

When I attempt to connect to the cluster every panel I visit has:

```
Connection Error
auth error: auth exec command '"oci" "ce" "cluster" "generate-token" "--cluster-id" "<my-cluster-ocid>" "--region" "us-phoenix-1"' failed with status exit status: 1: Output { status: ExitStatus(unix_wait_status(256)), stdout: "", stderr: "Python path configuration:\n PYTHONHOME = '/tmp/.mount_aptakupv1ije/usr/'\n PYTHONPATH = '/tmp/.mount_aptakupv1ije/usr/share/pyshared/:'\n program name = '/usr/bin/python'\n isolated = 0\n environment = 1\n user site = 1\n import site = 1\n sys._base_executable = '/usr/bin/python'\n sys.base_prefix = '/tmp/.mount_aptakupv1ije/usr'\n sys.base_exec_prefix = '/tmp/.mount_aptakupv1ije/usr'\n sys.platlibdir = 'lib'\n sys.executable = '/usr/bin/python'\n sys.prefix = '/tmp/.mount_aptakupv1ije/usr'\n sys.exec_prefix = '/tmp/.mount_aptakupv1ije/usr'\n sys.path = [\n '/tmp/.mount_aptakupv1ije/usr/share/pyshared/',\n '',\n '/tmp/.mount_aptakupv1ije/usr/lib/python310.zip',\n '/tmp/.mount_aptakupv1ije/usr/lib/python3.10',\n '/tmp/.mount_aptakupv1ije/usr/lib/python3.10/lib-dynload',\n ]\nFatal Python error: init_fs_encoding: failed to get the Python codec of the filesystem encoding\nPython runtime state: core initialized\nModuleNotFoundError: No module named 'encodings'\n\nCurrent thread 0x00007f03ec65cc40 (most recent call first):\n <no Python frame>\n" }
```

Octant and Openlens connect fine, as well as kubectl on the command line. I don't have another OS with access to this cluster to test on further.

**Comment by goenning** [comment]
Hey, thanks for the bug report @Whoops  

Looks like an issue with our AppImage. `oci` uses Python to generate tokens for kubernetes and AppImage does some trickery with python environment variables, causing Aptakube not to find the python Modules oci needs.

I will have a fix for this very shortly, would you like to try? If you could join our discord (https://discord.gg/FFp9bxHxtx) and DM there it would help a lot as I can share the new AppImage with you.

**Comment by goenning** [comment]
This should be fixed now with 1.4.3, but please reopen if it's not!
